stream-chat-android-client/src/main/java/io/getstream/chat/android/client/parser/JsonUtils.kt

```diff
@@ -0,0 +1,39 @@
+package io.getstream.chat.android.client.parser
+
+import com.google.gson.ExclusionStrategy
+import com.google.gson.FieldAttributes
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import io.getstream.chat.android.core.internal.InternalStreamChatApi
+
+@InternalStreamChatApi
+public object JsonUtils {
+
+    public fun defaultStreamGson() : Gson {
```

Comment:
This could be a lazy property within this class, so that there's literally just one instance of Gson in the app.

Reply:
Makes sense, let's not have multiple instances of gson